src/components/NgoList/index.css

```diff
@@ -0,0 +1,40 @@
+:root {
+  --main-ngoTable-bg-color: #eff4f6;
+  --ngoTable-text-color: #002736;
+  --orange-link: #f2c94c;
+  --dark-orange: #d88315;
+  --orange: #fc9e01;
+}
+
+.ngoList {
+  width: 100%;
+  padding: 0 6rem 0 2rem;
```

Comment:
Why are there zeros for padding?

Reply:
because I just need the padding for right and left so instead of writing padding-left and padding-right, I used padding